src/utils/useTransactionHistory.ts

```
@@ -0,0 +1,309 @@
+              );
+              mintsInfo.set(instruction.parsed.info.mint, mintAccount.value);
+            }
+            if (
+              instruction.parsed.info?.mint &&
+              mintsInfo.get(instruction.parsed.info.mint)?.data
+            ) {
+              const mintData = mintsInfo.get(instruction.parsed.info.mint)
+                ?.data as ParsedAccountData;
+              if (mintData.parsed?.info?.extensions) {
+                for (const ext of mintData.parsed?.info?.extensions) {
+                  if (ext.extension === "tokenMetadata") {
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)